src/styles/_global.scss

```diff
@@ -0,0 +1,399 @@
+@import "variables";
+
+@import url('https://fonts.googleapis.com/css?family=Open+Sans:400,600,700&display=swap');
```

Comment:
This shouldn't be necessary, as the $font should be this: https://github.com/turtlequeue/website/blob/6a12e74c6373a0a75ba16e4c3c4f9158667686c0/src/App.js#L37

Reply:
This was the font from the new design. Also, as a structure of style changed, the font is was not in "styled-components".

Reply:
I see, I had a note in the brief about the typography - but fine I'll change it myself.